core/lib/zksync_core/src/proof_data_handler/request_processor.rs

```diff
@@ -118,13 +119,31 @@ impl RequestProcessor {
             }
         };
 
+        let storage_batch = self
```

Comment:
nit(optional): you can add a separate method to load only `pubdata_input` as `l1_batch` has some other heavy columns